tf-101/5-creating.md

```diff
@@ -25,48 +25,47 @@ redirect: https://developer.oracle.com/tutorials/tf-101/5-creating/
 {% slides %}
 {% imgx aligncenter assets/terraform-101.png 400 400 "Terraform 101" "Terraform 101 Tutorial Series" %}
 
-The next few lessons in this tutorial will take you through an adventure where resources are created, modified, and then finally destroyed using Terraform.  While you got a little taste for this in the [Experiencing Terraform lesson](2-experiencing-terraform.md), we'll cover the topics in greater detail here, pointing out more details along the way.
+The next few lessons in this tutorial will guide you through an adventure where resources are created, modified, and then finally destroyed using Terraform. While you got a little taste for this in the [Experiencing Terraform lesson](2-experiencing-terraform.md), we'll cover the topics in greater detail here, pointing out more details along the way.
 
-Let's start using Terraform on Oracle Cloud Infrastructure (OCI) by deploying a Virtual Cloud Network (VCN), a foundational OCI resource in which you can deploy other OCI resources to.
+Let's start using Terraform on Oracle Cloud Infrastructure (OCI) by deploying a virtual cloud network (VCN), a foundational OCI resource in which you can deploy other OCI resources to.
 
 ## Prerequisites
 
-You should have an Oracle Cloud Infrastructure (OCI) account setup.  [Click here]({{ site.urls.alwaysfree }}) to create a new cloud account.
+You should have an OCI account setup. [Click here]({{ site.urls.alwaysfree }}) to create a new cloud account.
```

Comment:
Only looking at a diff here not the entire file, but if OCI isn't already expanded on first use it should be. But that might have already happened, in which case ignore this comment.

Reply:
Yup, it was on line 30